api/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your models here.
+ACTIVITY_LEVEL = (
+			("SC", "Sleepy_Cat"),
+			("LC", "Lazy_Cat"),
+			("AC", "Active_Cat"),
+			("MC", "Mentor_Cat"),
+	)
+
+class UserProfile(models.Model):
+	user = models.OneToOneField(User) 
+	#id = models.AutoField(primary_key=True) 
+	fullname = models.CharField(max_length=100, blank=False)
```

Comment:
Same applies as in previous comment. "SC" is a bad variable to compare against. "inactive" would be better and that would visually map to "Sleepy Cat".